scripts/README.md

```diff
@@ -16,26 +16,22 @@ on the command line and follow the instructions.
 Create 32 samples using command line arguments:
 
 ```shell
-generate_waveforms \
+generate-waveforms \
   --hypocentral_distance 531 \
   --magnitude 6 \
   --vs30 154 \
   --hypocentre_depth 10.0 \
   --azimuthal_gap 130 \
   --num_samples 32 \
-  --output waveforms.h5 \
-  --edm_checkpoint ../weights/edm.ckpt \
-  --autoencoder_checkpoint ../weights/autoencoder.ckpt
+  --outfile waveforms.h5

```

Comment:
Why are you removing the checkpoints?

Reply:
okay, then we should also remove the info with downloading the weights, cause this makes it just confusing. i would personally rather leave this in

Reply:
also, if we never provide it, we assume the user has a possibility to download the weights when they use the script. it is better to leave that to the user...

Reply:
Mmmh, but this would force the user to be able to download the weights automatically (like on the train). I think we should give the option and show how the method is called.